src/frontends/pytorch/src/op/avg_poolnd.cpp

```diff
@@ -22,7 +22,14 @@ OutputVector translate_avg_poolnd(const NodeContext& context) {
     num_inputs_check(context, 6, 7);
     auto input = context.get_input(0);
     auto kernel = context.const_input<Shape>(1);
-    auto strides = context.const_input<Strides>(2);
```

Comment:
is `strides` always constant if provided? Can it be output of some operation?

Reply:
It can, but we didn't met such models.